### PR TITLE
Added DL cases for Javascript

### PR DESCRIPTION
**Pull request by aastjais**
Add non compliant and compliant samples for Javascript detectors.

Note: All the test cases have 100% recall and precision.

### CODE REVIEW

**Review by saurabh-amazon — APPROVED** [review]
Reviewed all changes. Changes look fine. Approved.

File: javascript/src/detectors/javascript-do-not-store-sensitive-data-in-local-storage/javascript-do-not-store-sensitive-data-in-local-storage_compliant.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=javascript-do-not-store-sensitive-data-in-local-storage@v1.0 defects=0}
+function conforming(userId, username) {
+    //Compliant: Sensitive data is not stored in `localStorage`.
```
Comment (grl-wang) [inline javascript/src/detectors/javascript-do-not-store-sensitive-data-in-local-storage/javascript-do-not-store-sensitive-data-in-local-storage_compliant.js:6]: pls update the compliant comment format to `// Compliant: `

File: javascript/src/detectors/javascript-do-not-store-sensitive-data-in-local-storage/javascript-do-not-store-sensitive-data-in-local-storage_non-compliant.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=javascript-do-not-store-sensitive-data-in-local-storage@v1.0 defects=1}
+
+function nonCompliant(username, password) {
+    localStorage.setItem("USERNAME", username);
+    //Noncompliant: Sensitive data being stored in `localStorage`.
```
Comment (grl-wang) [inline javascript/src/detectors/javascript-do-not-store-sensitive-data-in-local-storage/javascript-do-not-store-sensitive-data-in-local-storage_non-compliant.js:8]: pls update the Noncompliant comment format to `// Noncompliant: `

File: javascript/src/detectors/javascript-express-jwt-not-revoked/javascript-express-jwt-not-revoked_compliant.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+// SPDX-License-Identifier: MIT-0
+
+// {fact rule=javascript-express-jwt-not-revoked@v1.0 defects=0}
+
+const express = require('express');
+const isRevoked = require('./isRevoked');
+const jwt = require('express-jwt');
+
+app2.route('/someURL/:id').get((req, res) => {
+    // Compliant: JWT token is not revoked.
```
Comment (grl-wang) [inline javascript/src/detectors/javascript-express-jwt-not-revoked/javascript-express-jwt-not-revoked_compliant.js:11]: is this a valid compliant code example

**Review by saurabh-amazon — APPROVED** [review]
Reviewed corrections and changes. Approved.

**Review by grl-wang — APPROVED** [review]

**Pull request merged**